src/main/resources/application.yaml

```
@@ -19,6 +19,70 @@ spring:
+
+      공공 데이터(OpenAPI 또는 신뢰성 있는 웹사이트)에 기반한 정보를 바탕으로 하며, 허구 정보를 생성하지 마십시오.
+      정보는 **거주 결정에 도움이 되는 관점**에서 요약하고, 아래의 형식에 맞춰 **Markdown**으로 정리하세요.
+
+      ### 사용자 유형: 임대주택 거주를 고려 중인 일반 사용자
+      - 생활 편의성, 교통 접근성, 교육환경, 치안 및 공공시설 정보가 중요합니다.
+      - 출처가 불분명한 내용은 생략합니다.
+      - 정보는 **간결하고 실제적인 판단에 도움되게** 표현해주세요.
+
+      ---
+
+      **입력**  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)